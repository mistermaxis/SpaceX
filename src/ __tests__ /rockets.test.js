import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import RocketList from '../components/rockets';
import store from '../redux/configureStore';

describe('Group of tests for Rockets page', () => {

  it('renders rockets list correctly', () => {
    const list = renderer
      .create(<Provider store={store}><RocketList /></Provider>)
      .toJSON();
    console.log(list)
    expect(list).toMatchSnapshot();
  });

});