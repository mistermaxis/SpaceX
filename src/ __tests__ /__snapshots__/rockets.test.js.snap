// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Group of tests for Rockets page renders rockets list correctly 1`] = `
<div
  className="container-fluid"
>
  ;
</div>
`;
